src/Maintenance/Services/LogInfoRetrieverService.php

```diff
@@ -40,34 +46,48 @@ public function retrieveInfo($value)
 
         $files = $this->filterLogFilesByDate($value, $files);
 
+        if (!isset($_SERVER['REQUEST_URI'])) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_SERVER Superglobal detected.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)